builtin/mainmenu/tab_local.lua

```diff
@@ -52,6 +52,11 @@ if enable_gamebar then
 						core.set_topleft_text(pkgmgr.games[j].name)
 						core.settings:set("menu_last_game",pkgmgr.games[j].id)
 						menudata.worldlist:set_filtercriteria(pkgmgr.games[j].id)
+						--switch game sounds, if found
+						core.sound_stop_all()
+						local _,_,technical_game_name = string.find(pkgmgr.games[j].gamemods_path, ".*\\(.*)\\.*")
+						core.sound_play(technical_game_name, true)
+						core.log("Playing sound: " .. technical_game_name)
```

Comment:
Why not `pkgmgr.games[j].id`? That should already be the technical name.

Reply:
Originally it was because I name my games all with upper case starting letters, but this is not how contentDB works and is bad practice on my part. I will switch to pkgmgr.games[j].id